### PR TITLE
[16.0][FIX] product_packaging_container_deposit: Deprecated method invalida…

### PR DESCRIPTION
**Pull request by bosd**
…te_cache()

### CODE REVIEW

**Review by Dranyel-Bosd — APPROVED** [review]
LGTM :+1:

**Comment by bosd** [comment]
@rafaelbn Can you please review?

**Comment by bosd** [comment]
@rafaelbn Can you have a look at this one? Merge if oke :hushed:

**Comment by rafaelbn** [comment]
I think @santostelmo and @simahawk who made PRs previously are better in context to review this technical one. 😄 🙏🏼

File: sale_product_packaging_container_deposit/tests/test_sale_product_packaging_container_deposit.py [hunk 1]
```diff
@@ -267,7 +267,7 @@ def test_sale_product_packaging_container_deposit_quantities_case7(self):
         box_line = self.sale_order.order_line.filtered(
             lambda ol: ol.product_id == self.box
         )
-        (box_line | pallet_line).invalidate_cache()
+        (box_line | pallet_line).invalidate_model()
```
Comment (simahawk) [inline sale_product_packaging_container_deposit/tests/test_sale_product_packaging_container_deposit.py]: Shouldn't be `invalidate_recordset`?

Comment (santostelmo) [reply sale_product_packaging_container_deposit/tests/test_sale_product_packaging_container_deposit.py]: I would go for `invalidate_recordset`

> Invalidate the cache of all records of ``self``'s model, when the cached values no longer correspond to the database values.

But I am fine with `invalidate_model` as the result will be the same.

**Review by santostelmo — APPROVED** [review]

File: sale_product_packaging_container_deposit/tests/test_sale_product_packaging_container_deposit.py (hunk 1 above)
Comment (bosd) [reply sale_product_packaging_container_deposit/tests/test_sale_product_packaging_container_deposit.py]: Changed it to `invalidate_recordset` and force pushed.
Ready for merge now.

**Review by jbaudoux — APPROVED** [review]

**Pull request merged**